### DISCUSSION TITLE
Retrobytes -  Let's Make a DOS BBS in a offensively modern way

### DISCUSSION THREAD

**Discussion by stuaxo** [dosemu2/dosemu2#2189] [General]
As a fan of anachronism, this is really enjoyable 
https://www.youtube.com/watch?v=wLVHXn79l8M

It builds on https://github.com/jgoerzen/docker-bbs-renegade (the dosemu here is pretty old, which is my one complaint), but then goes completely over the top adding all of the modern scalable stuff like Kubernetes etc.

(If I ever get round to learning Kubernetes this may even be the video I start with).

Since the BBS isn't publicly launched yet, maybe there is time for the version of DOSEMU running to be updated before then.

**Comment by stsp** [comment]
At 16:00 he says he got a working
locking... on dosemu1? Did it ever
work there? Quite strange.

> Since the BBS isn't publicly launched yet, maybe there is time for the version of DOSEMU running to be updated before then.

There seems to be a lot of BBS-related
business around dosemu, but everyone
seems to have some problems installing
bbs with dosemu2.
Maybe you can try these things out and
see if it is really problematic, or not...

**Reply by stuaxo** [reply]
Re: dosemu1/2.. it's not possible to say until the next video comes out: while the original repo is has old stuff, it's possible he's updated those bits; we will see once the then.

I'm definitely going to have a go once that comes out, as while I know docker already - I need to get experience with all the other infrastructure - but doing it purely for some webapp is just too much like my actual work,  but containerising and automating dos apps is the right kind of fun.

I need to do a from-scratch reinstall of dosemu2 first, I had an error the other day when I tried updating for the first time in a while, but I need to make sure it's not my setup + if I have the same issue, report it on it's own ticket.

EDIT: That sorted things out.

**Reply by stuaxo** [reply]
Oh I wonder if the setup is more based on this, which is in fact dosemu2
https://retroshed.us/dos-bbs-on-linux/

I'll have a look..

**Reply by stsp** [reply]
Would be great!
We need to sort this BBS stuff out!
All instructions about setting up BBS
with dosemu2 are either quite crappy
(have lots of legacy config setting etc),
or people just give up saying "this doesn't
work". We need to release our own
BBS instructions, But I'll never get to this.
